src/Controller/SeasonController.php

```diff
@@ -33,6 +36,20 @@ public function index()
                 impressed with Colbert’s beard and the manly way he pretends to smoke a pipe.",
             ]
         ];
-        return $this->twig->render('Season/season.html.twig', ['episodes' => $episodes]);
+        $season = new SeasonManager();
+        $season = $season->selectOneById($id);
+        $tvshow = new SeasonManager();
+        $tvshow = $tvshow->getShowBySeason($id);
```

Comment:
Je pense que tu peux appeler une seule fois le SeasonManager ici, et ensuite lui demander tes méthodes : 
```suggestion
        $seasonManager = new SeasonManager();
        $season = $seasonManager->selectOneById($id);
        $tvshow = $tseasonManager->getShowBySeason($id);
```

Reply:
corrected